experimental/parser/testdata/parser/package/too_big.proto

```diff
@@ -0,0 +1,18 @@
+// Copyright 2020-2024 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto2";
+
+package thispackagenameconsistsof513bytes.
```

Comment:
It would be good to also test a case that interstitial whitespace to be ignored in between components and separators. Here's an existing test cases for the existing (non-experimental) parser impl:
https://github.com/bufbuild/protocompile/blob/main/parser/validate_test.go#L454-L457